monitoring/uss_qualifier/webapp/README.md

```diff
@@ -1,7 +1,7 @@
 # RID Qualifier WebApp
```

Comment:
Is this app still known as RID Qualifier or it should be changed to `USS Qualifier`?

Reply:
Should be USS Qualifier -- thanks!